src/functions/IndGraph.jl

```diff
@@ -0,0 +1,83 @@
+# indicator of an affine set
+
+"""
+  IndGraph(A::Array{Real,2})
+
+Returns the function `g = ind{(x, y) : Ax = y}`.
```

Comment:
Hmmm... I doubt that. Unless the KKT system remains quasidefinite. If I have some time I will check the math.